src/UI/Icon.elm

```diff
@@ -443,6 +445,26 @@ remove hint =
     Icon (Properties hint Remove) defaultOptions
 
 
+{-| A trash can with an "x" on it.
+
+    Icon.trash "Delete item"
+
+-}
+trash : String -> Icon
```

Comment:
Icons are named after their action, so we don't misuse it to designate two different actions.
In this case, I prefer it named `Delete`.